java/android/nnstreamer/src/androidTest/java/org/nnsuite/nnstreamer/APITestCommon.java

```diff
@@ -42,6 +99,9 @@ public static void initNNStreamer() {
                 fail();
             }
         }
+
+        /* Copy nnstreamer folder in assets to the app's files directory */
+        copyAssetFolder(getContext(), "nnstreamer", getContext().getFilesDir().getAbsolutePath() + "/nnstreamer/");
```

Comment:
how about adding this in the block `if (!mInitialized) try`?, to copy once when called initNNStreamer()

Reply:
Cool. I updated the code as you suggested, PTAL